package.json

```diff
@@ -45,7 +45,7 @@
         "exponential-backoff": "^2.2.1",
         "form-data": "^2.5.1",
         "jszip": "^3.4.0",
-        "marked": "^1.0",
+        "markdown-it": "^11.0.0",
```

Comment:
Why do we have to switch to a different Markdown library? `marked` should support nested lists as you can see in the [demo](https://marked.js.org/demo/?text=%23%23%201.0.0%0A*%20**Preview%3A**%20%20A%20live%20display%20of%20the%20generated%20HTML%20as%20it%20would%20render%20in%20a%20browser.%0A%20%20%20%20*%20asd%0A%20%20%20%20%20%20%20%20*%20foo%0A*%20**HTML%20Source%3A**%20%20The%20generated%20HTML%20before%20your%20browser%20makes%20it%20pretty.%0A&options=%7B%0A%20%22baseUrl%22%3A%20null%2C%0A%20%22breaks%22%3A%20false%2C%0A%20%22gfm%22%3A%20true%2C%0A%20%22headerIds%22%3A%20true%2C%0A%20%22headerPrefix%22%3A%20%22%22%2C%0A%20%22highlight%22%3A%20null%2C%0A%20%22langPrefix%22%3A%20%22language-%22%2C%0A%20%22mangle%22%3A%20true%2C%0A%20%22pedantic%22%3A%20false%2C%0A%20%22sanitize%22%3A%20false%2C%0A%20%22sanitizer%22%3A%20null%2C%0A%20%22silent%22%3A%20false%2C%0A%20%22smartLists%22%3A%20false%2C%0A%20%22smartypants%22%3A%20false%2C%0A%20%22tokenizer%22%3A%20null%2C%0A%20%22walkTokens%22%3A%20null%2C%0A%20%22xhtml%22%3A%20false%0A%7D&version=master).

Reply:
I reinvestigated it and found that the formatting of marked confused me so hard, I assumed it does not support nested lists.